CMakeLists.txt

```diff
@@ -27,6 +27,73 @@ if(POLICY CMP0074)
     cmake_policy(SET CMP0074 NEW)
 endif()
 
+################################################################################
+# clang tools
+
+set(CLANG_TOOLS_SEARCH_PATHS
+    ${ClangTools_PATH}
+    $ENV{CLANG_TOOLS_PATH}
+    /usr/local/bin
+    /usr/bin
+    "C:/Program Files/LLVM/bin"
+    "$ENV{CONDA_PREFIX}/Library/bin"
+    "${HOMEBREW_PREFIX}/bin"
+)
+
+find_program(
+    CLANG_FORMAT
+    NAMES "clang-format"
```

Comment:
you should search also for names like `clang-format-8` and maybe rate this `clang-format-8` higher than `clang-format`
I installed clang format 8 for my ubuntu 16.04 and do not have a binary `clang-format`

```
find_program(
    CLANG_FORMAT
    NAMES 
        "clang-format-9"
        "clang-format-8"
        "clang-format"
    PATHS ${CLANG_TOOLS_SEARCH_PATHS}
    DOC "Path to clang-format"
)
```

Reply:
Will add `clang-format-8`